core/shared/src/main/scala/spire/syntax/Literals.scala

```diff
@@ -60,3 +62,13 @@ class EuLiterals(s:StringContext) {
   def big():BigInt = macro Macros.euBigInt
   def dec():BigDecimal = macro Macros.euBigDecimal
 }
+
+object primitives {
+  implicit class IntAs(val n:Int) extends AnyVal {
```

Comment:
We don't need `AnyVal` here, since the constructor call will be rewritten -- I would remove it.